libs/common/src/test-helpers/mock-data/fee-record-correction.entity.mock-builder.ts

```diff
@@ -6,11 +6,15 @@ import { RecordCorrectionReason, RequestedByUser } from '../../types';
 export class FeeRecordCorrectionEntityMockBuilder {
   private readonly correction: FeeRecordCorrectionEntity;
 
-  public constructor(correction?: FeeRecordCorrectionEntity) {
-    this.correction = correction ?? FeeRecordCorrectionEntityMockBuilder.forFeeRecord(new FeeRecordEntityMockBuilder().build()).correction;
+  private constructor(correction: FeeRecordCorrectionEntity) {
+    this.correction = correction;
   }
 
-  public static forFeeRecord(feeRecord: FeeRecordEntity): FeeRecordCorrectionEntityMockBuilder {
+  public static forIsCompleted(isCompleted: boolean): FeeRecordCorrectionEntityMockBuilder {
```

Comment:
this name doesn't make full sense? should be isCompleted?

Reply:
The preoperty on the entity is called 'isCompleted' and then the convention we have is an instantiation method starts with 'for' followed by the name of the property and then a method overwriting the default value the 'for' method sets for a specific property has the convention of starting with 'with'.

I agree this appears a little weird, and we could have an alternative convention for boolean fields if we wanted, but I don't think it should be just the property name, and I can't think of anything better than 'for' personally.

Reply:
This is always called as follows: `FeeRecordCorrectionEntityMockBuild.forIsCompleted(true)....` or `FeeRecordCorrectionEntityMockBuild.forIsCompleted(false)....` which I do think reads ok?

Reply:
I think they should all be changed at a later stage